apps/demos/package.json

```diff
@@ -126,7 +126,7 @@
         "stylelint-config-recommended-vue": "1.5.0",
         "stylelint-config-standard": "35.0.0",
         "systemjs-builder": "0.16.15",
-        "testcafe": "2.6.2",
+        "testcafe": "3.6.2",
```

Comment:
Can we separate the version update for the demos and the DevExtreme repository? In this case, the changes will be more obvious.